Linux/Deploy/Image/Dockerfile

```diff
@@ -77,6 +77,14 @@ RUN curl https://aka.ms/sqlpackage-linux -4 -sL -o '/tmp/sqlpackage.zip' \
   && sudo ln -sf $HOME/sqlpackage/sqlpackage /usr/local/bin
 ENV SqlPackage=/root/sqlpackage/sqlpackage
 
+RUN mkdir -p $HOME/azcopy \
```

Comment:
Could this follow a similar flow to SqlPackage install above? Should be able to be very similar other than the unzip/tar